packet_test.go

```diff
@@ -151,16 +150,16 @@ func TestParseIPLayer(t *testing.T) {
 	isIPv4 := true
 	ipv4Packet := gopacket.NewPacket(createMockmDNSPacket(isIPv4, true), decoder, options)
 
-	computedIPv4, computedIsIPv6 := parseIPLayer(ipv4Packet)
-	if !reflect.DeepEqual(dstIPv4Test, computedIPv4) || (computedIsIPv6 == isIPv4) {
+	computedIsIPv6 := parseIPLayer(ipv4Packet)
+	if computedIsIPv6 == isIPv4 {
```

Comment:
Let's replace `isIPv4` with `true` to make this test case a bit more readable

Reply:
I'm still not a big fan of this. I would like to add `assert` to the tests to make them even clearer in the future